docs/bundles/SyliusResourceBundle/update_resource.rst

```diff
@@ -136,6 +136,29 @@ You can also perform more complex redirects, with parameters. For example:
                     route: app_genre_show
                     parameters: { id: $genreId }
 
+[API] Returning resource or no content
+--------------------------------------
+
+Depending on your app approach it can be useful to return a changed object or only 204 HTTP code, which indicates that everything worked smoothly.
+By default a 204 code is returned. If you would like to receive a whole object as a response you should set a `return_content` option to true.
```

Comment:
No, it is not. By default, we do not return anything after updating. And I just didn't want to change current behaviour. The whole object is returned for `applyStateMachineTransitionAction` by default